sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -38,7 +44,7 @@ internal class NextLinkOperationImplementation : IOperation
             bool skipApiVersionOverride = false,
             string? apiVersionOverrideValue = null)
```

Comment:
If the caller is passing in both an `apiVersionOverride` value and a flag to `skipApiVersionOverride` - couldn't they just not pass the override if they wanted to skip?   Why do we need two parameters?

Reply:
API version removed